lib/manager/pre-commit/extract.ts

```diff
@@ -0,0 +1,138 @@
+import yaml from 'js-yaml';
+import { forEach } from 'traverse';
+import { id as githubTagsId } from '../../datasource/github-tags';
+import { id as gitlabTagsId } from '../../datasource/gitlab-tags';
+import { logger } from '../../logger';
+import { SkipReason } from '../../types';
+import { hasKey } from '../../util/object';
+import { regEx } from '../../util/regex';
+import { PackageDependency, PackageFile } from '../common';
+
+import { PrecommitGitDependency, matchesPrecommitGitHeuristic } from './util';
+
+function determineDatasource(
+  repository: string,
+  hostName: string
```

Comment:
Pass domainName instead, if that's what we're ultimately interested in

Reply:
hmm, it's more tricky to get the actual domain name. The domain name field in the regex actually just catches the `.com`, because of the eager matching.
I'll change the regex to actually match the domain name

Reply:
'k, I changed the regex to be able to directly ( `===` ) compare the domain name